components/LeagueCard/LeagueCard.tsx

```diff
@@ -23,7 +23,7 @@ const LeagueCard = React.forwardRef<HTMLAnchorElement, ILeagueCardProps>(
       data-testid="LeagueCard"
       href={href}
       className={cn(
-        'LeagueCard flex max-h-32 place-items-center gap-6 rounded-lg border bg-card p-4 text-card-foreground shadow-sm dark:border-zinc-800 hover:bg-zinc-800 transition',
+        'LeagueCard flex max-h-32 h-32 place-items-center gap-6 rounded-lg border bg-card p-4 text-card-foreground shadow-sm dark:border-zinc-800',
```

Comment:
DO NOT set a hard height on this component. This will remove its flexibility and create bugs when content overflows.

Reply:
@shashilo This is outdated, can it be resolved?

Reply:
@shashilo pinging you for a reminder, please advise.